RadClass/H0.py

```diff
@@ -44,7 +51,36 @@ def __init__(self, significance=0.05, gross=True, energy_bins=1000):
             # all p-vals for rejected hypothesis are saved in this sparse array
             self.triggers = np.empty((0, energy_bins+1))
 
+    def binom(self, x1, n, p):
+        '''
+        Private method for running binomial test.
+
+        Return:
+        lpval: log base 10 p-value result of scipy.stats.binomtest.
+        '''
+        # np.log10(1E-350), chosen to be smaller than any possible result
+        min_lpval = -350.0
+        # scipy.stats.binomtest will fail if n (# of trials)
+        # is less than 1 (possible for high-energy bins)
+        if int(n) < 1:
+            lpval = 0.0
+        else:
+            pval = stats.binomtest(int(x1), int(n), p,
+                                   alternative='two-sided').pvalue
+            if pval == 0.0:
+                lpval = min_lpval
+            else:
+                lpval = np.log10(pval)
```

Comment:
Putting the `np.log10` operation after checking for a float rounding avoids the occasional `RuntimeWarning` that happens when attempting `np.log10(stats.binomtest()=0.0)`